frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -220,8 +220,10 @@ const PlatformWrapper = ({
     return packageType === "advanced"
       ? `fleetctl package --type=YOUR_TYPE --fleet-url=${config?.server_settings.server_url} --enroll-secret=${enrollSecret} --fleet-certificate=PATH_TO_YOUR_CERTIFICATE/fleet.pem`
       : `fleetctl package --type=${packageType} ${
-          includeFleetDesktop ? "--fleet-desktop " : ""
-        }--fleet-url=${
+          config && !config.server_settings.scripts_disabled
+            ? "--enable-scripts "
+            : ""
```

Comment:
nit: if possible to avoid the negation here
```suggestion
          config && config.server_settings.scripts_disabled
            ? ""
            : "--enable-scripts "
```

Reply:
@roperzh, the component props allow for the config to be undefined so my thinking was that we probably want to exclude the flag if the config is undefined (e.g., in the edge case where the component is mounted before the config loads) as well as if the server setting is true, hence the somewhat awkward negative formulation.

Reply:
that makes sense, thanks for the context!